test/suite/test_config08.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_config08.py
+#   Test the configuration that enables/disables dirty tree flushing.
+#
+
+import time, wiredtiger, wttest
+from wtscenario import make_scenarios
+
+class test_config08(wttest.WiredTigerTestCase):
+
+    logging = [
+        ('log_off', dict(log='(enabled=false)')),
+        ('log_on', dict(log='(enabled=true)')),
+    ]
+    fileFlush = [
+        ('flush_off', dict(file_close_sync='false')),
+        ('flush_on', dict(file_close_sync='true')),
+    ]
```

Comment:
These should just contain the true/false and add in the config setting once. Also we do not usually use camelCase. See comment below about connection too.
```
logging = [
   ('log_off', dict(logging='false')),
        ('log_on', dict(logging='true)')),
    ]
    flush = [
        ('flush_off', dict(flush='false')),
        ('flush_on', dict(flush='true')),
    ]
```

Reply:
Fixed.